purchases/src/main/kotlin/com/revenuecat/purchases/common/diagnostics/DiagnosticsTracker.kt

```diff
@@ -42,6 +42,12 @@ internal class DiagnosticsTracker(
         const val VERIFICATION_RESULT_KEY = "verification_result"
         const val RESPONSE_TIME_MILLIS_KEY = "response_time_millis"
         const val PRODUCT_TYPE_QUERIED_KEY = "product_type_queried"
+        const val PRODUCT_ID_KEY = "product_id"
+        const val OLD_PRODUCT_ID_KEY = "old_product_id"
+        const val HAS_INTRO_TRIAL = "has_intro_trial"
+        const val HAS_INTRO_PRICE = "has_intro_price"
```

Comment:
This is a nit about naming, but some of these end with `KEY` and other don't. Is there some specific reason?

Reply:
I missed that! Will fix, thank you!